src/main.rs

```diff
@@ -26,9 +28,10 @@
         feature = "native-tls"
     )),
 ))]
-compile_error!(
+/*compile_error!(
     "exactly one of the features \"native-roots\", \"webpki-roots\" or \"native-tls\" must be enabled"
-);
+);*/
+print!("hi");
```

Comment:
Please revert

Reply:
Yea I forget to delete that and expect code, I used them to ignore the errors msg from clippy and lsp, Sorry again I'll fix it.